doc/1-intro/glossary.md

```diff
@@ -72,6 +72,8 @@
 
 **LFO**: low frequency oscillator. a wave with a slow period (often below hearing range) used to alter other sounds.
 
+**LFSR**: linear-feedback shift register. an algorithm of generating pseudo-random noise that loops, also known as "periodic noise". depending on the location of the two "taps" used, different lengths of noise loops are generated.
```

Comment:
1. There may be more than two taps.
2. Taps aren't described.
3. Improve wording in "an algorithm of generating". Maybe "a method to generate"?